### PR TITLE
Don't bother hashing the mapping, use UID

### PR DESCRIPTION
**Pull request by esoergel**
@czue

### CODE REVIEW

**Comment by czue** [comment]
looks good, will wait for tests

File: corehq/apps/hqcase/management/commands/ptop_preindex.py [hunk 1]
```diff
@@ -86,7 +86,8 @@ def handle(self, *args, **options):
             print ("Reindexing master pillows that do not exist yet "
                    "(ones with aliases skipped)")
 
-            preindexable_pillows = filter(pillow_index_exists, aliasable_pillows)
+            preindexable_pillows = [pillow for pillow in aliasable_pillows
+                                    if not pillow_index_exists(pillow)]
```
Comment (esoergel) [inline corehq/apps/hqcase/management/commands/ptop_preindex.py]: I think this was why the preindex didn't happen

File: corehq/apps/app_manager/management/commands/ptop_fast_reindex_apps.py [hunk 2]
```diff
@@ -9,3 +9,4 @@ class Command(ElasticReindexer):
     doc_class = ApplicationBase
     view_name = 'app_manager/applications'
     pillow_class = AppPillow
+    default_chunk_size = 1000
```
Comment (esoergel) [inline corehq/apps/app_manager/management/commands/ptop_fast_reindex_apps.py]: Sending 10,000 apps to ES in one go surpasses ES's default 100MB limit locally.  Probably not a great idea on prod either.

File: corehq/apps/hqcase/management/commands/ptop_preindex.py (hunk 1 above)
Comment (czue) [inline corehq/apps/hqcase/management/commands/ptop_preindex.py]: :+1: thanks for fixing!

**Comment by czue** [comment]
late to the party but this looks great - thanks!

**Pull request merged**